cmake/Modules/BuildCommon.cmake

```diff
@@ -463,6 +463,8 @@ function(add_katana_sphinx_target target_name)
   if (BUILD_DOCS STREQUAL "internal")
     set(sphinx_options "-W;${sphinx_options}")
   endif ()
+  # Enable parallel reading / writing, 4 workers gave optimal time reduction
+  set(sphinx_options "-j 4;${sphinx_options}")
```

Comment:
Is `-j 4` correct? I feel like it should be `-j;4` so the two arguments are split at the CMake list level.

BTW, where did you measure the time that showed 4 workers was best? And how different was, say, 2 and 6 workers? I ask because if it's really sensitive to this parameter we might need to make it configurable so CI can use an optimized value that's different from the local build defaults.

Reply:
- `-j 2` yielded 138s
  - `-j 4` yielded 137s
  - `-j 6` yielded 142s
  - `-j 8` yielded 157s

I measured this in my shell.